src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final short ELEMENT_INDEX = 0;
+    private static final short VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File file = new File(fromFileName);
+        int buy = 0;
+        int supply = 0;
+        int result = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
```

Comment:
```suggestion
        int buy = 0;
        int supply = 0;
        int result = 0;
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
```
There is no need to create separate object